kerby-kerb/kerb-client/src/main/java/org/apache/kerby/kerberos/kerb/client/KrbClientBase.java

```diff
@@ -271,15 +271,22 @@ public void storeTicket(TgtTicket tgtTicket,
      */
     public void storeTicket(SgtTicket sgtTicket, File ccacheFile) throws KrbException {
         LOG.info("Storing the sgt to the credential cache file.");
-        if (!ccacheFile.exists()) {
+        boolean isFreshNew = !ccacheFile.exists() || (ccacheFile.length() == 0);
+        if (isFreshNew) {
             createCacheFile(ccacheFile);
         }
         if (ccacheFile.exists() && ccacheFile.canWrite()) {
-            CredentialCache cCache = new CredentialCache();
+            
```

Comment:
There is some trailing whitespace here, and in a few other places as well that could be removed.

Reply:
I'm going to commit the changes, removed the blank rows.. I left only the blank rows that were in the original code. I was a bit confused cause I don't see a clear code format policy  in your code. Sometimes you have empty rows around important "if" blocks. I think this is right and I performed the same by adding blank lines to improve readability... Have you a place where I can read your code formatting policies?

Reply:
Ok, I totally misunderstood. Trailing spaces are not black lines! :-( English is not my natural language and sometimes I fail. I adde a new commit to the pull request, deleted the trailing spaces and re-added few blank lines around if/else blocks.. that.. I think ... improve readability.
Regards